### PR TITLE
Added the ability to create college majors

### PR DESCRIPTION
**Pull request by herdingbats**
I needed my fake data to include field of study, so I made a module for it. It's a bit idiosyncratic (the list of fields comes from the University of Chicago) but it's a start.

### CODE REVIEW

**Comment by fcurella** [comment]
I think this could go into the `profile` provider, rather than being its own

File: faker/providers/major/__init.py__ [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		"Social Sciences",
+		"Sociology",
+		"South Asian Languages and Civilizations",
+		"Statistics",
+		"Theater and Performance Studies",
+		"Tutorial Studies",
+		"Visual Art"
+	)
+
+def major(self):
```
Comment (fcurella) [inline faker/providers/major/__init.py__]: I think you missed an indentation level here

Comment (herdingbats) [reply faker/providers/major/__init.py__]: Will fix and re-submit.

Comment (herdingbats) [reply faker/providers/major/__init.py__]: In the larger scheme of things, would it make sense to have a few different functions for education? One for level (from "some HS" up through "PhD") and then one for field (from "broad field" to "detailed field" to "major"--I'm pulling these categories from p. 2 of this Census report: https://www.census.gov/prod/2012pubs/acs-18.pdf.

**Comment by fcurella** [comment]
closing for inactivity

**Pull request closed without merging**